### PR TITLE
Update erlang.mk and deps

### PR DESCRIPTION
**Pull request by elbrujohalcon**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,45 +2,29 @@ PROJECT = canillita
...
-test-build-plt: ERLC_OPTS=$(TEST_ERLC_OPTS)
-test-build-plt:
-	@$(MAKE) --no-print-directory test-dir ERLC_OPTS="$(TEST_ERLC_OPTS)"
-	$(gen_verbose) touch ebin/test
-
-plt-all: PLT_APPS := $(ALL_TEST_DEPS_DIRS)
-plt-all: test-deps test-build-plt plt
-
-dialyze-all: app test-build-plt dialyze
+SHELL_OPTS = -s sync -config ${CONFIG} -s lager
```
Comment (harenson) [inline Makefile]: Remove `lager` from here or add it in the `SHELL_DEPS` list.

File: test/canillita_newsitems_SUITE.erl [hunk 2]
```diff
@@ -164,34 +164,34 @@ not_found(_Config) ->
...
-   , <<"body">> := <<"body1">>
+   , <<"title">> := <<"titlenf1">>
+   , <<"body">> := <<"bodynf1">>
    , <<"created_at">> := _CreatedAt
    } = sr_json:decode(Body1),
 
   ct:comment("Unable to get non-existing newsitem with a valid newspaper"),
   #{status_code := 404} =
-    canillita_test_utils:api_call(get, NewsItem1Url ++ "/non-existing-id"),
+    canillita_test_utils:api_call(get, NewsItem1Url  ++  "/non-existing-id"),
```
Comment (harenson) [inline test/canillita_newsitems_SUITE.erl]: There are some extra spaces here.

File: test/canillita_newsitems_SUITE.erl [hunk 3]
```diff
@@ -164,34 +164,34 @@ not_found(_Config) ->
...
 
   ct:comment("Unable to get existing newsitem id with invalid newspaper"),
   #{status_code := 404} =
     canillita_test_utils:api_call(
       get
-    , "/newspapers/non-existing-newspaper/news/" ++ NewsItem1Id
+    , "/newspapers/non-existing-newspaper/news/"  ++  NewsItem1Id
```
Comment (harenson) [inline test/canillita_newsitems_SUITE.erl]: There are some extra spaces here.

File: Makefile (hunk 1 above)
Comment (elbrujohalcon) [inline Makefile]: Added to `SHELL_DEPS`, instead

**Pull request merged**